test/passing/refs.default/exp_grouping.ml.ref

```diff
@@ -375,13 +375,14 @@ let x =
       match Tbl.find dist_tbl (pv1, pv2) with
       | None ->
           (* FIXME: temporary hack to avoid Jane Street's annoying warnings. *)
-          begin [@warning "-3"]
-            try
-              let path', dist = Dijkstra.shortest_path pgraph pv1 pv2 in
-              let path = unwrap_path path' in
-              Tbl.set dist_tbl ~key:(pv1, pv2) ~data:(path, dist);
-              Some (path, dist)
-            with Not_found | Not_found_s _ -> None
+          begin
+          [@warning "-3"]
+          try
```

Comment:
I feel like this could be fixed by checking whether there are attributes in the special case for `Pexp_match`.